cluster/addons/podsecuritypolicies/restricted.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: restricted-v1
+  annotations:
+    kubernetes.io/description: 'This policy demonstrates best practices for
+      running Pods with minimal security priviliges.'
+    # To use custom seccomp or apparmor profiles, make a copy of this policy and
+    # add the custom profiles here.
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default'
+    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'docker/default'
+    apparmor.security.beta.kubernetes.io/defaultProfileName:  'runtime/default'
+    apparmor.security.beta.kubernetes.io/allowedProfileNames: 'runtime/default'
+  labels:
+    kubernetes.io/cluster-service: 'true'
+    addonmanager.kubernetes.io/mode: Reconcile
+spec:
+  privileged: false
+  allowPrivilegeEscalation: false
```

Comment:
Do we need to also specify `DefaultAllowPrivilegeEscalation: false` here?

Reply:
No, it's defaulted from `allowPrivilegeEscalation` here: https://github.com/kubernetes/kubernetes/blob/98ed5dd8a26929245e4aa30f110242ac7bb7cbbd/pkg/security/podsecuritypolicy/provider.go#L192